openebs/openebs-provisioner.go

```diff
@@ -281,3 +291,41 @@ func GetStorageClassName(options controller.VolumeOptions) *string {
 	}
 	return options.PVC.Spec.StorageClassName
 }
+
+// parseClassParameters extract the new fstype other then "ext4"(dafault) which
+// can be changed via "openebs.io/fstype" key and env OPENEBS_VALID_FSTYPE
+func parseClassParameters(params map[string]string) (string, error) {
+	var fsType string
+	for k, v := range params {
+		switch strings.ToLower(k) {
+		case "openebs.io/fstype":
+			fsType = v
+		}
+	}
+	if len(fsType) == 0 {
+		fsType = defaultFSType
+	}
+
+	//Get openebs supported fstype from ENV variable
+	validENVFSType := os.Getenv("OPENEBS_VALID_FSTYPE")
```

Comment:
Wondering if this could go into somewhere else like init block. Otherwise, this logic will run whenever there is a volume provision. The outcome will remain same in every execution.

Reply:
@princerachit i think this approach is good because user may require different FSTYPE for different  apps otherwise he needs to run another instance of openebs-provisioner in order to provision volume in the other requested format.Using this approach he can modify the env as per the requirement. 
@AmitKumarDas  @kmova what's your view on this ?

Reply:
The `OPENEBS_VALID_FSTYPE` contains a comma separated supported fstypes. How is it going to change? The env var is intrinsic to openebs provisioner container. It has nothing to do with applications.

Reply:
Yes, to modify it in a running provisioner pod we have to use it here.

Reply:
Why/how would we modify `OPENEBS_VALID_FSTYPE` ?

Reply:
We can export it inside the provisioner pod.

Reply:
Ok. So I had a discussion with @prateekpandey14 regarding the above issue. We are doing this to give the user(or the devops admin) control over what file systems could be created using openebs. This has an obvious caveat that wrong or invalid values in `OPENEBS_VALID_FSTYPE` will break formatting. However, the onus to ensure that correct values are passed as well as used lies on the devops admin. Approving this PR now.